docs/azure/sdk/configure-proxy.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Configure a proxy when using the Azure SDK for .NET 
+description: Learn different approaches for configuring a proxy for use with the Azure SDK for .NET client libraries.
+ms.topic: conceptual
+ms.custom: devx-track-dotnet, engagement-fy23
+ms.date: 11/15/2024
+---
+
+# Configure a proxy when using the Azure SDK for .NET
+
+If your organization requires the use of a proxy server to access Internet resources, some configuration is required to use the Azure SDK for .NET client libraries. Once configured, the proxy is applied to the underlying `HttpClient` instance used for HTTP operations.
+
+The proxy can be configured via code or via an environment variable. The approach you choose depends on the desired behavior. Set the appropriate environment variable if you want the proxy to apply globally to all service clients created within the current process. Alternatively, configure the proxy via code to selectively apply the settings to service clients.
+
+> [!IMPORTANT]
+> The following instructions apply only to [libraries with a dependency on Azure.Core](protocol-convenience-methods.md#azure-sdk-client-library-dependency-patterns).
```

Comment:
👍

We can remove/modify this note once https://github.com/Azure/azure-sdk-for-net/issues/47192 is completed, depending on the outcome there.